### PR TITLE
[glossary]Add "Child Universe"

### PR DESCRIPTION
**Pull request by CrystalBallBe**

### CODE REVIEW

File: _data/version2-english/glossary.yml [hunk 1]
```diff
@@ -8,7 +8,17 @@ All Theoretical REP: |-
...
   </ol>
   This value is used to calculate the <a href="#Forking_Threshold">forking threshold</a>.
-Child Universe: <i>under construction</i>
+Child Universe: |-
+  A child universe is a  <a href="#Universe">universe</a> spawned by its <a href="#Parent_Universe">parent universe</a> when a market in the parent universe <a href="#Fork">forks</a>.
+  <br>
+  <br>
+  In a child universe, the <a href="#Forking_Market">forking market</a> is <a href="#Finalized_Market">finalized</a> as the <a href="#Outcome">outcome</a> that aligns with that particular child universe. For example, a <a href="#Market">market</a> has 3 possible outcomes: <code>Yes</code>, <code>No</code>, and <code>Invalid</code>. When the market forks, 3 new universes are created: <code>universe Yes</code>, <code>universe No</code>, and <code>universe Invalid</code>. In each universe, its corresponding outcome becomes the <a href="#Final_Outcome">final outcome</a>, such as in the <code>universe Yes</code> the final outcome of the forking market is <code>Yes</code>, in the <code>universe No</code> the final outcome of the forking market is <code>No</code>.
```
Comment (MicahZoltu) [inline _data/version2-english/glossary.yml:15]: Consider putting each sentence on a separate line.  When it is rendered, it should all render as a paragraph if you don't include `<br>` or `<p>` or multiple newlines, but it makes the PR much easier to review.

**Comment by MicahZoltu** [comment]
Happen to have a preview of this one?

File: _data/version2-english/glossary.yml (hunk 1 above)
Comment (CrystalBallBe) [reply _data/version2-english/glossary.yml:15]: From now on I will do that 🌝

**Comment by CrystalBallBe** [comment]
This is the preview:
https://crystalballbe.github.io/version2-english/7-glossary.html#Child_Universe

**Comment by CrystalBallBe** [comment]
> Consider putting each sentence on a separate line. When it is rendered, it should all render as a paragraph if you don't include `<br>` or `<p>` or multiple newlines, but it makes the PR much easier to review.

In order to reflect this suggestion, I'm closing this pull request and resubmit.

**Pull request closed without merging**